src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/custom.py

```diff
@@ -265,8 +262,6 @@ def update_pool(client, pool_id, json_file=None, command_line=None,  # pylint:di
     except (ValidationError, ClientRequestError) as ex:
         raise CLIError(ex)
 
-update_pool.__doc__ = PoolUpdatePropertiesParameter.__doc__ + "\n" + StartTask.__doc__
```

Comment:
No one will intend to directly call the this custom function. The docstring here is only used for build up the cli param help string. So the broken doc string doesn't bother any real user.